spec/mailers/collection_invitation_mailer_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CollectionInvitationMailer do
+  describe '#send_invitation' do
+    subject(:invitation_email) { described_class.with(collection:, recipient: user).send_invitation }
+
+    let(:collection) { create(:collection) }
+    let(:user) { create(:user) }
+
+    it 'sends an email to the correct recipient' do
+      expect(invitation_email.to).to include(user.email)
+    end
+
+    it 'has the correct subject' do
+      expect(invitation_email.subject).to include(collection.title)
+    end
+
+    it 'contains the correct content' do
+      expect(invitation_email.body.encoded).to include(user.name)
+      expect(invitation_email.body.encoded).to include(collection.title)
+      expect(invitation_email.body.encoded).to include('collaborate')
```

Comment:
I was not sure what we were testing here. Part of the invitation message? Can we check the hole thing?

```suggestion
      expect(invitation_email.body.encoded).to include(I18n.t('collections.invitation_mailer.invitation_message'))
```

Reply:
Yes, I just checked some keywords of the message and used the specs from another mailer as a template. Your idea seems fine, too!